.docs/Token-and-IDs.md

```diff
@@ -1,10 +1,11 @@
 # Obtaining Token and Channel IDs
 
-> **Warning**:
-> **Do not share your token!**
-> A token gives full access to an account. To reset a user token, change your account password. To reset a bot token, click on [Regenerate](#how-to-get-a-bot-token) in the bot settings.
+> [!WARNING]
+> **Do not share your token!** A token gives full access to an account.  
+> To reset a user token, change your account password.  
```

Comment:
```suggestion
> **Do not share your token!** A token gives full access to an account.
> To reset a user token, change your account password.
```

Still a few trailing spaces here

Reply:
Trailing spaces (line break) on warning are necessary for better formatting:

Without them:
<img width="1042" alt="image" src="https://github.com/user-attachments/assets/8203a5f7-5a0d-4dd4-a238-00f24cde1069" />

With:
<img width="915" alt="image" src="https://github.com/user-attachments/assets/dbfb64d6-564e-4843-8a5d-20740dc7ea92" />

<hr>

This
```
> [!WARNING]
> **Do not share your token!** A token gives full access to an account.
> 
> To reset a user token, change your account password.
>
> To reset a bot token, click on [Reset Token](#how-to-export-with-a-bot-token) in the bot settings.
```

Renders as
<img width="562" alt="image" src="https://github.com/user-attachments/assets/ce8f8db3-2eb6-416d-902a-4cbb9ae116da" />

So not ideal